src/FrontendAccountCreation.Web/Controllers/ReprocessorExporter/ApprovedPersonController.cs

```diff
@@ -260,30 +316,65 @@ public async Task<IActionResult> CheckYourDetails()
             return View(viewModel);
         }
 
+        [HttpGet]
+        [Route(PagePath.ApprovedPersonPartnershipCanNotBeInvited)]
+        [OrganisationJourneyAccess(PagePath.ApprovedPersonPartnershipCanNotBeInvited)]
+        public async Task<IActionResult> PersonCanNotBeInvited([FromQuery] Guid id)
+        {
+            var session = await _sessionManager.GetSessionAsync(HttpContext.Session);
+            SetBackLink(session, PagePath.ApprovedPersonPartnershipCanNotBeInvited);
+            await _sessionManager.SaveSessionAsync(HttpContext.Session, session);
+            
+            return View(new LimitedPartnershipPersonCanNotBeInvitedViewModel { Id = id });
+        }
+
+        [HttpPost]
+        [Route(PagePath.ApprovedPersonPartnershipCanNotBeInvited)]
+        [OrganisationJourneyAccess(PagePath.ApprovedPersonPartnershipCanNotBeInvited)]
+        public IActionResult PersonCanNotBeInvited(LimitedPartnershipPersonCanNotBeInvitedViewModel model)
+        {
+            return RedirectToAction("CheckYourDetails", "AccountCreation");
```

Comment:
Can we use check your details of ReEx present on this controller rather than account creation